docs/source/format/Flight.rst

```diff
@@ -333,9 +333,13 @@ schemes for the given transports:
 +----------------------------+--------------------------------+
 | (reuse connection)         | arrow-flight-reuse-connection: |
 +----------------------------+--------------------------------+
-| UCX_ (plaintext)           | ucx:                           |
+| UCX_ (plaintext) (1)       | ucx:                           |
 +----------------------------+--------------------------------+
 
+Notes:
+
+* \(1) Flight UCX transport has been deprecated on the 19.0.0 release.
```

Comment:
Can we add a link to https://arrow.apache.org/docs/dev/format/DissociatedIPC.html ?

Reply:
Done, see: http://crossbow.voltrondata.com/pr_docs/45080/format/Flight.html#location-uris